internal/cli/streams/instance/create_test.go

```diff
@@ -27,32 +27,71 @@ import (
 	atlasv2 "go.mongodb.org/atlas-sdk/v20231115008/admin"
 )
 
+const (
+	testProjectID = "create-project-id"
+)
+
 func TestCreateOpts_Run(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	mockStore := mocks.NewMockStreamsCreator(ctrl)
 
-	buf := new(bytes.Buffer)
-	opts := &CreateOpts{
-		store:    mockStore,
-		name:     "ExampleStream",
-		provider: "AWS",
-		region:   "VIRGINIA_USA",
-	}
-	opts.ProjectID = "create-project-id"
-
-	expected := &atlasv2.StreamsTenant{Name: &opts.name, GroupId: &opts.ProjectID, DataProcessRegion: &atlasv2.StreamsDataProcessRegion{CloudProvider: "AWS", Region: "VIRGINIA_USA"}}
-
-	mockStore.
-		EXPECT().
-		CreateStream(opts.ProjectID, expected).
-		Return(expected, nil).
-		Times(1)
-
-	if err := opts.Run(); err != nil {
-		t.Fatalf("Run() unexpected error: %v", err)
-	}
-	t.Log(buf.String())
-	test.VerifyOutputTemplate(t, createTemplate, expected)
+	t.Run("stream instances create", func(t *testing.T) {
+		buf := new(bytes.Buffer)
+		opts := &CreateOpts{
+			store:    mockStore,
+			name:     "ExampleStream",
+			provider: "AWS",
+			region:   "VIRGINIA_USA",
+		}
+		opts.ProjectID = testProjectID
+
+		expected := &atlasv2.StreamsTenant{Name: &opts.name, GroupId: &opts.ProjectID, DataProcessRegion: &atlasv2.StreamsDataProcessRegion{CloudProvider: "AWS", Region: "VIRGINIA_USA"}}
```

Comment:
should expected have SP30 as tier here?

Reply:
Pending discussion on the other comment, the backend will fill out this value. So this test should expect the field not to be completed. 

If we decide the atlas cli should be the one defaulting the value, we can update both tests to expect the field.

Reply:
Both tests now expect tier to be fulfilled; done here! [df8d5c2](https://github.com/mongodb/mongodb-atlas-cli/pull/2839/commits/df8d5c23b9f4736fe11de5ae054df1b209a6c150)